app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt

```diff
@@ -48,8 +48,7 @@ class DashboardActivity : ComponentActivity() {
             Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU -> {
                 arrayOf(
                     permission.READ_MEDIA_IMAGES,
-                    permission.READ_MEDIA_AUDIO,
-                    permission.READ_EXTERNAL_STORAGE        // TODO: Remove this if not needed.
```

Comment:
Where did you find the doc for this? I know we discussed this before but a doc/article link would be good here.

Reply:
READ_MEDIA_AUDIO was removed by mistake, but I tested the removal of READ_EXTERNAL_STORAGE and it works fine!

Reply:
It would be great if we had a doc to support this. I want to get rid of these as well but as you yourself mentioned before, don't remove them if it doesn't break.

Reply:
That quote tho :P Ill look for the docs.

Reply:
Funnily enough, we don't even need permission for images because if our app is the one who writes the images to mediastore (no matter the location of the image), I'm able to access the image because it exists because of our app in the first place. m! Android